ludwig/api.py

```diff
@@ -766,6 +771,9 @@ def on_epoch_end(self, trainer, progress_tracker, save_path):
                 # Ensure model weights are saved to the driver if training was done remotely
                 if self.backend.is_coordinator() and not skip_save_model:
                     self.model.save(model_dir)
+                    if save_ludwig_config_with_model_weights:
+                        weights_save_path = os.path.join(model_dir, MODEL_WEIGHTS_FILE_NAME, "ludwig_config.json")
```

Comment:
@sanjaydasgupta Do we want a typehint here for `weights_save_path` (if it has not been declared before) as well?  Thank you.